### DISCUSSION TITLE
[vite-node] [ERR_LOAD_URL] /.nuxt/fetch.mjs

### DISCUSSION THREAD

**Discussion by Gabrielmoraesp** [nuxt/nuxt#25955] [Questions]
All my projects in nuxt 3 and new ones that I try to create, the low error appears. Has anyone ever experienced this?

![image](https://github.com/nuxt/nuxt/assets/440323/cf65f89f-72a8-4521-b8a9-8d2e2816494b)

**Comment by AleksandrCherednichenko-developer** [comment]
@Gabrielmoraesp The same thing is happening with my nuxt project, I have been trying to overcome this problem for the second day. I've seen comments that this is solved by upgrading the nuxt version to newer ones, but in my case it doesn't work. Please let us know if you find a solution
